pkgs/development/python-modules/torchaudio/default.nix

```diff
@@ -31,27 +34,42 @@ buildPythonPackage rec {
     TORCH_CUDA_ARCH_LIST = "${lib.concatStringsSep ";" torch.cudaCapabilities}";
   };
 
+  # https://github.com/pytorch/audio/blob/v2.1.0/docs/source/build.linux.rst#optional-build-torchaudio-with-a-custom-built-ffmpeg
+  FFMPEG_ROOT = symlinkJoin {
+    name = "ffmpeg";
+    paths = [
+      ffmpeg-full.bin
+      ffmpeg-full.dev
+      ffmpeg-full.lib
+    ];
+  };
+
   nativeBuildInputs = [
     cmake
     pkg-config
     ninja
   ] ++ lib.optionals cudaSupport [
     cudaPackages.cuda_nvcc
   ];
+
   buildInputs = [
+    ffmpeg-full
     pybind11
+    sox
   ] ++ lib.optionals cudaSupport [
     cudaPackages.libcurand.dev
     cudaPackages.libcurand.lib
     cudaPackages.cuda_cudart # cuda_runtime.h and libraries
     cudaPackages.cuda_cccl.dev # <thrust/*>
     cudaPackages.cuda_nvtx.dev
     cudaPackages.cuda_nvtx.lib # -llibNVToolsExt
+    cudaPackages.cudatoolkit
```

Comment:
Marking as a place `cudatoolkit` was added

Reply:
Just removed it.